### ISSUE TITLE
CSS Preprocessors

### ISSUE THREAD

**Issue by OSP123** [OSP123/post-grad-curriculum#37]

**Comment by OSP123** [comment]
Going to go with LESS and SASS (with SCSS).

**Comment by oshalygin** [comment]
consider PostCSS, its the future.  SASS and LESS are becoming less and less mainstream with all the bloat and footguns.

**Comment by OSP123** [comment]
I was actually wondering about this. I used to use SASS at last company, but that was Rails. Is PostCSS the new things being used nowadays?

**Comment by oshalygin** [comment]
Yeah PostCSS gives you the modularity that you need.  Most people don't need all of the features from SASS but they do like certain things like variables and nesting.  PostCSS gives you the ability to plug and play without bringing in a whole beast.

It's more or less become the defacto standard on modern React applications.  That said, you'll find differing opinions :)

https://github.com/postcss/postcss

**Comment by OSP123** [comment]
Nice, appreciate the input :). I'll take a look at it and then probably do a combo of the three, emphasizing postcss for React and what you mentioned.

**Comment by OSP123** [comment]
Going to try and implement this: https://blog.logrocket.com/getting-started-with-postcss-in-2019-484262a4d725
